src/test/storage/counting_quotient_filter_test.cpp

```diff
@@ -0,0 +1,54 @@
+#include <memory>
+#include <set>
+#include <string>
+#include <unordered_set>
+#include <utility>
+#include <vector>
+
+#include "../base_test.hpp"
+#include "gtest/gtest.h"
+
+#include "../lib/storage/base_column.hpp"
+#include "../lib/storage/chunk.hpp"
+#include "../lib/statistics/chunk_statistics/counting_quotient_filter.hpp"
+#include "../lib/types.hpp"
+
+namespace opossum {
+
+class CountingQuotientFilterTest : public BaseTest {
```

Comment:
Could you please also test the other configurations besides `remainder_bits == 8`?

Concerning the probabilistic tests: I totally see your point. I don't like it either. But I don't see a way we can be sufficiently sure that this third-party library (that nobody after you will be able to understand) and your bindings are working as expected, apart from thorough testing.

Also the card estimation tests are too few in my opinion.

Reply:
Now testing all remainder configurations and integers as well as strings.